docs/scripting/api-environment.md

```diff
@@ -5,6 +5,7 @@ tags:
     - stable
 ---
 
+Minecraft Bedrock Edition uses a their own version of JavaScript, which is based on QuickJS. It uses the ECMAScript module (ESM) system for organizing and loading code, which allows for a more modular and organized approach to writing scripts for the game.
```

Comment:
Minecraft: Bedrock Edition uses their own version of JavaScript based on QuickJS. It uses the ECMAScript module (ESM) system for organizing and loading code, which allows for a more modular and organized approach to writing scripts for the game.